tests/foreman/ui/test_activationkey.py

```diff
@@ -270,7 +270,7 @@ def test_negative_create_with_invalid_name(self):
                         name=name,
                         env=ENVIRONMENT,
                     )
-                    self.assertIsNotNone(self.products.wait_until_element(
```

Comment:
it doesn't matter what object to use for `wait_until_element` as all of them has same implementation for that method (inherited from same class)